### ISSUE TITLE
Chat - System redirected you to LHN, NOT to the chat

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12597]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Issue found when executing #12169 
## Action Performed:
1. Open two sessions one as User A and one as User B
2. Make sure User B is navigate to the chat with User A and save the URL
3. Close User B tab (or kill app)
4. Create message as User A on chat with User B and wait 10 seconds
5. Open tab (or app) as User B using the URL saved from the second step (on native use a deep link)

## Expected Result:
The system should redirected you to the chat

## Actual Result:
System redirected you to LHN

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- iOS
- Android
- Mobile Web

**Version Number:** 1.2.25-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/200881797-75a234f0-f0f9-4d8e-91a4-da034e1c78e4.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** Applause internal team
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by zanyrenney** [comment]
Will check this out tomorrow.

**Comment by zanyrenney** [comment]
I don't understand this bug or the reproduction steps, please can you rephrase them or explain in more detail? To me, this looks like it is working as expected. @kavimuru

**Comment by zanyrenney** [comment]
bump @kavimuru

**Comment by zanyrenney** [comment]
bumping @kavimuru

**Comment by kavimuru** [comment]
@zanyrenney Sorry for the late reply. This is failure of the PR #12169 

> New message marker set correctly when navigating to directly to unread chat
> 
> 1. Open two sessions one as User A and one as User B
> 2. Make sure User B is navigate to the chat with User A and save the URL
> 3. Close User B tab (or kill app)
> 4. Create message as User A on chat with User B and wait 10 seconds
> 5. Open tab (or app) as User B using the URL saved from the second step (on native use a deep link)
> 6. See that new marker is set correctly
> 7. Verify that the chat in the LHN is marked as read
Step 5 is failing in this part

**Comment by zanyrenney** [comment]
No worriesm, thanks for letting me know! I'll try to reproduce tomorrow.

**Comment by jasperhuangg** [comment]
Hey hey happy to help out here, I was able to reproduce. You actually don't need to do all those steps:
1. Open the app on mobile web on a device with a small screen.
2. Paste in the URL to any report.
3. It navigates you to the LHN instead.
